### PR TITLE
companies filters

### PR DESCRIPTION
**Pull request by pro-mohamed-hamada**

### CODE REVIEW

File: app/Http/Controllers/BranchController.php [hunk 1]
```diff
@@ -41,17 +41,17 @@ public function store(BranchStoreRequest $request)
             $toast = [
                 'type' => 'success',
                 'title' => 'success',
-                'message' => trans('app.receiver_created_successfully')
+                'message' => trans('app.success_operation')
             ];
-            return to_route('companies.edit', $branchDTO->company_id)->with('toast',$toast);
```
Comment (eslam-eng) [inline app/Http/Controllers/BranchController.php:46]: why remove :  return to_route('companies.edit', $branchDTO->company_id)->with('toast',$toast); 
and convert dto to arry to get company_id also you have company id direct ?

File: app/Http/Controllers/BranchController.php [hunk 2]
```diff
@@ -41,17 +41,17 @@ public function store(BranchStoreRequest $request)
...
         } catch (Exception $e) {
             DB::rollBack();
             $toast = [
                 'type' => 'error',
                 'title' => 'success',
-                'message' => trans('app.receiver_created_successfully')
+                'message' => trans('app.success_operation')
             ];
-            return to_route('companies.edit', $branchDTO->company_id)->with('toast',$toast);
+            return to_route('companies.edit', Arr::get($branchDTO->toArray(), 'company_id'))->with('toast',$toast);
```
Comment (eslam-eng) [inline app/Http/Controllers/BranchController.php:54]: revert to this line : return to_route('companies.edit', $branchDTO->company_id)->with('toast',$toast);

File: resources/views/layouts/dashboard/branches/create.blade.php [hunk 3]
```diff
@@ -8,15 +8,6 @@
...
     <!-- Row -->
     <div class="row">
-        @if ($errors->any())
-            <div class="alert alert-danger">
-                <ul>
-                    @foreach ($errors->all() as $error)
-                        <li>{{ $error }}</li>
-                    @endforeach
-                </ul>
-            </div>
```
Comment (eslam-eng) [inline resources/views/layouts/dashboard/branches/create.blade.php:18]: revert this :  @if ($errors->any())
            <div class="alert alert-danger">
                <ul>
                    @foreach ($errors->all() as $error)
                        <li>{{ $error }}</li>
                    @endforeach
                </ul>
            </div>

File: resources/views/layouts/dashboard/companies/components/_filters.blade.php [hunk 4]
```diff
@@ -13,18 +13,44 @@
...
+                                    <div class="col-lg">
+                                        <livewire:location.cities/>
+                                    </div>
+                                    <div class="col-lg">
+                                        <livewire:location.areas/>
                                     </div>
+                                </div>
 
+                                <div class="row row-sm mt-4 mb-4">
+                                    <div class="col-lg">
```
Comment (eslam-eng) [inline resources/views/layouts/dashboard/companies/components/_filters.blade.php:26]: remove email filter

File: resources/views/layouts/dashboard/branches/create.blade.php (hunk 3 above)
Comment (pro-mohamed-hamada) [reply resources/views/layouts/dashboard/branches/create.blade.php:18]: done

File: resources/views/layouts/dashboard/companies/components/_filters.blade.php (hunk 4 above)
Comment (pro-mohamed-hamada) [reply resources/views/layouts/dashboard/companies/components/_filters.blade.php:26]: done

File: app/Http/Controllers/BranchController.php (hunk 1 above)
Comment (pro-mohamed-hamada) [reply app/Http/Controllers/BranchController.php:46]: not work

File: app/Http/Controllers/BranchController.php (hunk 2 above)
Comment (pro-mohamed-hamada) [reply app/Http/Controllers/BranchController.php:54]: not work

**Pull request merged**